server/templates/index.html

```diff
@@ -2,5 +2,5 @@
 {% block content %}
 <div id="page"></div>
 {{ page_data|json_script:"pageData" }}
-<script src="/server.home.dev.js"></script>
+<script src="/server.home.dev.js""></script>
```

Comment:
two quotes in a row after 'js' looks fishy... should that be:
```suggestion
<script src="/server.home.dev.js"></script>
```

Reply:
Whoops, accidental typo from something I was working on.